NetKAN/B9AnimationModules.netkan

```diff
@@ -9,7 +9,7 @@
     "license"        : "LGPL-3.0",
     "release_status" : "stable",
     "resources": {
-        "homepage" : "http://forum.kerbalspaceprogram.com/threads/92630"
+        "homepage" : "https://forum.kerbalspaceprogram.com/index.php?/topic/155491-*"
```

Comment:
Could this be changed to `?showtopic=155491`?  I know they both work but I prefer this format as it doesn’t have the weird bit that gets ignored at the end.

Reply:
Absolutely!